irobot_benchmark/package.xml

```diff
@@ -11,11 +11,13 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
 
   <build_depend>rclcpp</build_depend>
+  <build_depend>irobot_events_executor</build_depend>
```

Comment:
this dependency will likely break the CI

Reply:
Maybe related to https://github.com/irobot-ros/ros2-performance/pull/131#discussion_r1557743720 , should we still point to the irobot_events_executor after the upstream version [gets merged into rclcpp:irobot/humble](https://github.com/irobot-ros/rclcpp/pull/145) ?